samples/multi-assembly/pom.xml

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Thanks for adding a sample :+1: . Could you please add a README.md file too regarding build instructions? Something like we have in JIB sample: https://github.com/fabric8io/docker-maven-plugin/tree/master/samples/spring-boot-with-jib

Reply:
You're welcome! I've added a README as requested.

Reply:
Thanks a lot!